src/tools/compiletest/src/runtest.rs

```diff
@@ -2382,19 +2383,31 @@ impl<'test> TestCx<'test> {
         }
     }
 
-    /// Runs RMC on the test file specified by `self.testpaths.file`.
-    /// An error message is printed to stdout if verfication fails.
+    /// Runs RMC on the test file specified by `self.testpaths.file`. An error
+    /// message is printed to stdout if verfication result is not expected.
     fn run_rmc_test(&self) {
         // Other modes call self.compile_test(...). However, we cannot call it here for two reasons:
         // 1. It calls rustc instead of RMC
         // 2. It may pass some options that do not make sense for RMC
         // So we create our own command to execute RMC and pass it to self.compose_and_run_compiler(...) directly.
         let mut rmc = Command::new("rmc");
+        rmc.args(&self.props.rmc_flags)
```

Comment:
Please add a short comment about this arguments extension.

Reply:
Done. Please take a look.